src/openlrr/game/object/ObjectRecall.cpp

```diff
@@ -22,27 +25,163 @@ LegoRR::ObjectRecall_Globs & LegoRR::objectRecallGlobs = *(LegoRR::ObjectRecall_
 #pragma region Functions
 
 // <LegoRR.exe @00459450>
-//void __cdecl LegoRR::ObjectRecall_StoreMiniFigure(const LegoObject* liveObj);
+void __cdecl LegoRR::ObjectRecall_StoreMiniFigure(const LegoObject* liveObj)
+{
+
+	//Check type is correct, ability flags are set, and name is custom
+	if ((liveObj->type == LegoObject_MiniFigure) &&
+		(((liveObj->abilityFlags != ABILITY_FLAG_NONE || (liveObj->objLevel != 0)) || (liveObj->customName != nullptr)))) {
```

Comment:
Ghidra tries to religiously put parenthesis around if statement comparisons. Usually most of them are redundant, so try to clean those up if it isn't too much of a headache.

If you think parenthesis around a comparison makes it easier to read then keep them. For example, always use parenthesis to separate `&&` chains from `||` chains, even if order of operations doesn't require it.
e.g. `if ((conditionA && conditionB) || conditionC || conditionD) {`